### ISSUE TITLE
Forms.GL doesn't exit debugging on close

### ISSUE THREAD

**Issue by VahnRPG** [sqrMin1/MonoGame.Forms#11]
Using Visual Studio 2015
OS: Win 7
Can provide other stats if necessary

I was having issues getting forms.dx to work so I switched to forms.gl and that seemed to help. Unfortunately, when I attempt to close the form the debugger remains running until I manually stop the process in VS. Attached is a fresh winforms project with just forms.gl installed from nuget and a single draw window on a form and it consistently presents the same issue. I've tested this on multiple pcs and each one has the same issue.

[WindowsFormsApplication1.zip](https://github.com/sqrMin1/MonoGame.Forms/files/2492870/WindowsFormsApplication1.zip)

**Comment by sqrMin1** [comment]
I pushed a fix. The debugger should now close after exiting a MonoGame.Forms.GL application.

But please tell me what didn't work with the MonoGame.Forms.DX library. Because this should work on your windows machine.

Have you tried running the MonoGame.Forms.Tests.DX project?
Do you get an exception?

**Comment by VahnRPG** [comment]
Thanks for that! Lookin good now! :)

The DX issue I hadn't reported yet because I wasn't sure if it was my stuff or what, but I'll give a rundown of what was going on.

I started a new project, added DX via nuget, copied the drawtest, and it was working fine. I started copying over stuff from a pre-Forms winform that I was working on that had a layout I wanted to use. I c&p'ed it over, put the drawtest into a panel & set it to dock-fill. Seemed to be going okay so I added some more of the code I'd been working on and tried to get it running.

At some point it decided it no longer wanted to work as whenever I moved my mouse over the drawwindow it crashed the program with https://pastebin.com/BHXNajbU for an error. About then I noticed it wasn't displaying the "Hello Monogame.Forms" text any more, either. It was displaying the same stuff as in the designer view with the logo and the value in the Text field. I added some console logging and realized it wasn't calling the Form_Load I'd set up or the Initialize on the drawtest class.

Unfortunately I've since deleted that code when I got the GL version going, so I can't provide it as an example, sorry :/

**Comment by VahnRPG** [comment]
Actually while I was trying that right now I think I may have figured out at least part of the problem. When I was working on that I wasn't sure which version I should've been using, and I kept installing and reinstalling the 3 different versions on nuget.

I remembered this because I was trying to switch my project from the GL version to the DX version and upon doing so it completely messes up.........something....in the process that requires me to archive that version and roll back to a previous version in a new folder because the current one refuses to compile regardless of what may have done in an attempt to switch back.

And an attempt just now I realized I'd been installing the DX version then uninstalling the GL version, but apparently that wipes out the Monogame.Framework part. Uninstalling the GL version and installing the DX version seemed to work as expected. So, like I said, it was probably more my ignorance than your code that was an issue :)
